api/controllers/FlairController.js

```diff
@@ -95,9 +95,9 @@ module.exports = {
       var flagged = _.reject(flairs.fcs, Flairs.validFC);
       var ipAddress = req.headers['x-forwarded-for'] || req.ip;
       // Get IP matches with banned users
-      var banned_alts = await Event.find({content: {contains: ipAddress}, user: {not: req.user.name}}).then(function (eventResults) {
+      var banned_alts = _.map(await Event.find({content: {contains: ipAddress}, user: {not: req.user.name}}).then(function (eventResults) {
```

Comment:
This line is really confusing. Can you split it up a bit more?

Also, why do you keep doing .then(Function)? The whole point of await is we don't need to do that.